Mathlib/Data/Nat/Factors.lean

```diff
@@ -83,6 +83,11 @@ theorem primeFactorsList_prime {p : ℕ} (hp : Nat.Prime p) : p.primeFactorsList
   have : Nat.minFac p = p := (Nat.prime_def_minFac.mp hp).2
   simp only [this, primeFactorsList, Nat.div_self (Nat.Prime.pos hp)]
 
+theorem primeFactorsList_prime_iff {p : ℕ} : p.primeFactorsList = [p] ↔ p.Prime := by
```

Comment:
I'm not entirely sure about this name either; maybe `primeFactorsList_eq_singleton_iff` or `primeFactorsList_eq_singleton_iff_prime`… Though those are getting long. (Probably `primeFactorsList_prime` should be `primeFactorsList_of_prime`)